webapp/templates/proxy-create.xhtml

```diff
@@ -0,0 +1,303 @@
+        <div class="row">
+            <div class="col-sm-6">
+                <fieldset>
+                    <div class="form-group">
+                        <label for="local">Local name</label>
+                        <div>
+                            <input type="text" id="local" value="{rdfs:label}" class="form-control" required="required" autofocus="autofocus"
+                                onchange="calli.updateProperty(event, 'rdfs:label')" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label for="comment">Comment</label>
+                        <div>
+                            <textarea id="comment" class="form-control"
+                                onchange="calli.updateProperty(event, 'rdfs:comment')">{rdfs:comment}</textarea>
+                        </div>
+                    </div>
+                </fieldset>
+            </div>
+            <div class="col-sm-6">
+                <aside class="well">
+                    <p>PURLs are used to curate the URL resolution process, thus solving
+                    the problem of transitory URIs in location-based URI schemes like HTTP.</p>
+                    <p>Partial PURLs are a special class of PURL that must end with a '/' and 
+                    allow a hierarchy of URLs to be mapped to another resource. The request pattern
+                    may include regular expressions in the path subsequent to the PURL name.</p>
+                    <p>Other PURLs cannot end with a '/' and the request pattern only applies to the query string.</p>
+                </aside>
+            </div>
+        </div>        
+                
+        <div id="ruleOptions" style="display:none">
+            <!-- These are not used by the template but are required for validation when the resource is created -->
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/><!-- special case for viewing when no GET rule -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+        <input id="cacheProperty" type="hidden" property="calli:cacheControl" onchange="calli.updateProperty(event, 'calli:cacheControl')"/>
+                
+        <div class="row">
+            <div id="rules" class="col-sm-12">
+                <label>Handlers</label>
+
+                <div class="panel panel-default rule" data-method="GET">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">GET Handler
```

Comment:
I had considered this but deliberately decided not to do it because they are not like normal collapsible panels. We are not showing/hiding information we are actually adding/deleting handlers. I also didn't want to have a popup when a user collapses a panel to confirm that they are deleting the handler. I thought it was best to force the user to click + or - as a more deliberate action. What do you think? Would you still rather the header worked the same as the +/- button?